modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -208,101 +129,87 @@ LoadModule auth_gssapi_module modules/mod_auth_gssapi.so
 ----
 
-[discrete]
-[id="requestheader-master-ca-config_{context}"]
-===== Configuring the master
-
-The `identityProviders` stanza in the
-*_/etc/origin/master/master-config.yaml_* file must be updated as well:
-
-----
-  identityProviders:
-  - name: requestheader
-    challenge: true
-    login: true
-    provider:
-      apiVersion: v1
-      kind: RequestHeaderIdentityProvider
-      challengeURL: "https://[MASTER]/challenging-proxy/oauth/authorize?${query}"
-      loginURL: "https://[MASTER]/login-proxy/oauth/authorize?${query}"
-      clientCA: /etc/origin/master/proxyca.crt
-      headers:
-      - X-Remote-User
-----
-
-[discrete]
-===== Restarting services
-
-Finally, restart the following services:
-
+. Update the `identityProviders` stanza in the Custom Resource (CR):
++
 ----
-# systemctl restart httpd
-# master-restart api
-# master-restart controllers
+identityProviders:
+  - name: requestheaderidp
+    type: RequestHeader
+    requestHeader:
+      challengeURL: "https://<namespace_route>/challenging-proxy/oauth/authorize?${query}"
+      loginURL: "https://<namespace_route>/login-proxy/oauth/authorize?${query}"
+      ca:
+        name: ca-config-map
+        clientCommonNames:
+        - my-auth-proxy
+        headers:
+        - X-Remote-User
 ----
 
-[discrete]
-===== Verifying the configuration
+. Verify the configuration.
 
-. Test by bypassing the proxy. You should be able to request a token if you
-supply the correct client certificate and header:
+.. Bypass the proxy. You should be able to request a token if you supply the
+correct client certificate and header:
 +
 ----
 # curl -L -k -H "X-Remote-User: joe" \
    --cert /etc/pki/tls/certs/authproxy.pem \
-   https://[MASTER]:8443/oauth/token/request
+   https://<namespace_route>/oauth/token/request
 ----
 
-. If you do not supply the client certificate, the request should be denied:
+.. Exclude the client certificate from the previous command; the request should
+be denied:
 +
 ----
 # curl -L -k -H "X-Remote-User: joe" \
-   https://[MASTER]:8443/oauth/token/request
+   https://<namespace_route>/oauth/token/request
 ----
 
-. This should show a redirect to the configured `challengeURL` (with
+.. Execute this command to show a redirect to the configured `challengeURL` (with
 additional query parameters):
 +
 ----
 # curl -k -v -H 'X-Csrf-Token: 1' \
-   '<masterPublicURL>/oauth/authorize?client_id=openshift-challenging-client&response_type=token'
+   https://<namespace_route>/oauth/authorize?client_id=openshift-challenging-client&response_type=token
 ----
 
-. This should show a 401 response with a `WWW-Authenticate` basic challenge, a
-negotiate challenge, or both challenges:
+.. Run this command to show a 401 response with a `WWW-Authenticate` basic
+challenge, a negotiate challenge, or both challenges:
 +
 ----
 #  curl -k -v -H 'X-Csrf-Token: 1' \
-    '<redirected challengeURL from step 3 +query>'
+    <redirected challengeURL from step 3 +query>
```

Comment:
I'd replace this with an annotation about using the redirect URL that you obtained from the previous step.